### PR TITLE
feat: adding db.json data to marker on google maps

### PR DESCRIPTION
**Pull request by frank-mck**

### CODE REVIEW

File: client/src/components/maps/MyMap.js [hunk 1]
```diff
@@ -5,13 +5,11 @@ import { Hikes } from '../Hikes.js'
 import { Form } from '../Form.js'
 import { Button } from '../Button.js'
 import { GoogleMap, useLoadScript, Marker, InfoWindow } from "@react-google-maps/api"
-import * as hikeData from '../../dummyHikes.json'
+//import * as hikeData from '../../dummyHikes.json'
```
Comment (coo990) [inline client/src/components/maps/MyMap.js:8]: Are comments needed on an import that isn't being used?

File: client/src/components/maps/MyMap.js [hunk 2]
```diff
@@ -60,6 +69,9 @@ export const MyMap = () => {
   //   }
   // }
 
+  console.log(pins[0])
```
Comment (coo990) [inline client/src/components/maps/MyMap.js:72]: Is this printing anything on the webpage?

File: client/src/components/maps/MyMap.js [hunk 3]
```diff
@@ -73,13 +85,12 @@ export const MyMap = () => {
...
-      <Marker 
-        key={hike.id} 
-        position={hike.location} 
-      >
-      </Marker>
+      {pins.map((hike) => (
+        <Marker
+          key={ hike.id} 
+          position={{ "lat": parseFloat(hike.lat), "lng": parseFloat(hike.lng) }} 
+        >
```
Comment (coo990) [inline client/src/components/maps/MyMap.js:92]: Nice way to parse the data onto the markers

**Review by coo990 — APPROVED** [review]
Looks good, some comments of old code need to be removed if not used.

File: client/src/components/maps/MyMap.js (hunk 1 above)
Comment (mikey-roberts) [reply client/src/components/maps/MyMap.js:8]: Keeping temporarily, just until we know it's not needed and changes all working once PR accepted

File: client/src/components/maps/MyMap.js (hunk 2 above)
Comment (mikey-roberts) [reply client/src/components/maps/MyMap.js]: removed, looks like it was just for testing

**Comment by mikey-roberts** [comment]
LG2M

**Pull request merged**